modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -7,20 +7,17 @@
 
 Using {product-title} with Kuryr SDN has several known limitations.
 
-[discrete]
-[id="openstack-resource-limitations_{context}"]
-== {rh-openstack} resource limitations
-
-* An Amphora load balancer VM is deployed per OpenShift Service that uses the
-default Octavia load balancer driver (Amphora driver). Creating too many Services
-can cause you to run out of resources.
-
 [discrete]
 [id="openstack-version-limitations_{context}"]
 == {rh-openstack} version limitations
 
 Using {product-title} with Kuryr SDN has several limitations that depend on the {rh-openstack} version.
 
+* {rh-openstack} versions before 16 or without OVN octavia provider enabled uses
```

Comment:
This section seems largely to focus on RHOSP <16. Would it make sense for me to create a section for caveats that apply specifically to versions 13, 14, and 15 so that we don't need to keep stating "{rh-openstack} versions before 16..." here? @luis5tb

Reply:
not really. this paragraph is about OSP13, 14, 15. But also about OSP16 with neutron ml2/ovs, or OSP16 with ml2/ovn but without ovn-octavia enabled